<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_3D">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>40</y>
      <width>441</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="scarType_comboBox">
    <property name="geometry">
     <rect>
      <x>522</x>
      <y>41</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2D">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>99</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Histo_graphicsView">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>400</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="sliceSelectSlider">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>370</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="thresholdSelectSlider">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>670</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="showVolume_Button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Volume</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
